src/protobuf-net.Reflection/CSharpCodeGenerator.cs

```diff
@@ -911,6 +913,30 @@ private string GetTypeName(GeneratorContext ctx, string typeName)
             }
         }
 
+        private bool IsNullableType(GeneratorContext ctx, FieldDescriptorProto field)
+        {
+            if (ctx.IsEnabled("nullable") == false)
```

Comment:
``` c#
if (!ctx.IsEnabled("nullable"))
```

?

Reply:
In my experience people tend to overlook the !, that is why I have change into this way writing the statement. But I will of course fix it.